src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size = 0;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int index = getIndex(key);
+        checkCapacity(size, table);
```

Comment:
```suggestion
        checkCapacity(size);
```
table is a field of this class, so you can just access it from method